CONTRIBUTING.md

```diff
@@ -96,6 +96,12 @@ An installation of Node is required for development. If you don't have Node inst
 - https://stylelint.io/user-guide/integrations/editor
 - https://prettier.io/docs/en/editors.html
 
+The first time installing dependencies, you need to use the `--legacy-peer-deps` flag due to an ESLint dependency conflict. The conflict won't effect the components since ESLint is only in `devDependencies`.
```

Comment:
`effect` -> `affect`

Reply:
Do you think it's worth adding a note regarding this **_may_** no longer be needed once https://github.com/ionic-team/stencil-eslint supports ESLint 8?